x-pack/plugins/observability/public/components/shared/exploratory_view/series_builder/series_builder.tsx

```diff
@@ -0,0 +1,201 @@
+    { id: 'tpt', label: 'Throughput' },
+  ],
+  logs: [
+    {
+      id: 'logs',
+      label: 'Logs Frequency',
+    },
+  ],
+  metrics: [
+    { id: 'cpu', label: 'CPU usage' },
+    { id: 'mem', label: 'Memory usage' },
+    { id: 'nwk', label: 'Network activity' },
+  ],
+};
+
+export function SeriesBuilder() {
+  const { series, setSeries, allSeriesIds, removeSeries } = useUrlStorage(NEW_SERIES_KEY);
+
+  const { dataType, reportType, reportDefinitions = {}, filters = [] } = series;
+
+  const [isFlyoutVisible, setIsFlyoutVisible] = useState(!!series.dataType);
+
+  const { indexPattern } = useIndexPatternContext();
+
+  const getDataViewSeries = () => {
+    return getDefaultConfigs({
+      indexPattern,
+      reportType: reportType!,
+      seriesId: NEW_SERIES_KEY,
+    });
+  };
+
+  const columns = [
+    {
+      name: i18n.translate('xpack.observability.expView.seriesBuilder.dataType', {
+        defaultMessage: 'Data Type',
+      }),
+      width: '20%',
+      render: (val: string) => <DataTypesCol />,
+    },
+    {
+      name: i18n.translate('xpack.observability.expView.seriesBuilder.report', {
+        defaultMessage: 'Report',
+      }),
+      width: '20%',
+      render: (val: string) => (
+        <ReportTypesCol reportTypes={dataType ? ReportTypes[dataType] : []} />
+      ),
+    },
+    {
+      name: i18n.translate('xpack.observability.expView.seriesBuilder.definition', {
+        defaultMessage: 'Definition',
+      }),
+      width: '30%',
+      render: (val: string) =>
+        reportType && indexPattern ? (
+          <ReportDefinitionCol dataViewSeries={getDataViewSeries()} />
+        ) : null,
+    },
+    {
+      name: i18n.translate('xpack.observability.expView.seriesBuilder.filters', {
+        defaultMessage: 'Filters',
+      }),
+      width: '25%',
+      render: (val: string) =>
+        reportType && indexPattern ? <ReportFilters dataViewSeries={getDataViewSeries()} /> : null,
+    },
+    {
+      name: i18n.translate('xpack.observability.expView.seriesBuilder.breakdown', {
+        defaultMessage: 'Breakdowns',
+      }),
+      width: '25%',
+      field: 'id',
+      render: (val: string) =>
+        reportType && indexPattern ? (
+          <ReportBreakdowns dataViewSeries={getDataViewSeries()} />
+        ) : null,
+    },
+  ];
+
+  const addSeries = () => {
+    if (reportType) {
+      const newSeriesId = `${
+        reportDefinitions?.['service.name'] ||
+        reportDefinitions?.['monitor.id'] ||
+        ReportViewTypes[reportType]
+      }`;
+
+      const newSeriesN = {
+        reportType,
+        time: { from: 'now-30m', to: 'now' },
+        filters,
+        reportDefinitions,
+      } as SeriesUrl;
+
+      setSeries(newSeriesId, newSeriesN).then(() => {
+        removeSeries(NEW_SERIES_KEY);
+        setIsFlyoutVisible(false);
+      });
+    }
+  };
+
+  const items = [{ id: NEW_SERIES_KEY }];
+
+  let flyout;
+
+  if (isFlyoutVisible) {
```

Comment:
I don't believe you need this conditional. It appears as though the parent this is wrapped in already contains this conditional.